_shared/project/tests/conftest.py

```diff
@@ -0,0 +1,70 @@
+{% if cookiecutter.get("postgres") == "yes" %}
```

Comment:
This `conftest.py` file will only be rendered in newly created projects, `make template` won't create/update `conftest.py`.